tests/_support/Helper/Acceptance.php

```diff
@@ -30,6 +30,27 @@ public function seeFieldContains($I, $element, $value)
 		$this->assertNotFalse(strpos($I->grabValueFrom($element), $value));
 	}
 
+	/**
+	 * Helper method to enter text into a jQuery Select2 Field, selecting the option that appears.
+	 * 
+	 * @since 	1.9.6.4
+	 * 
+	 * @param 	AcceptanceTester 	$I
+	 * @param 	string 				$container 	Field CSS Class / ID
+	 * @param 	string 				$value 		Field Value
+	 * @param 	string 				$ariaAttributeName 	Aria Attribute Name (aria-controls|aria-owns)
+	 */
+	public function fillSelect2Field($I, $container, $value, $ariaAttributeName = 'aria-controls')
+	{
+		$fieldID = $I->grabAttributeFrom($container, 'id');
```

Comment:
Strange formatting here.

Reply:
Let me know what's strange with the formatting here.

Reply:
Lines 45-46 are more indented than the rest of the function. It looks like you're mixing tabs and spaces too.